core/src/banking_stage/transaction_scheduler/prio_graph_scheduler.rs

```diff
@@ -931,4 +931,107 @@ mod tests {
         assert_eq!(scheduling_summary.num_unschedulable, 0);
         assert_eq!(collect_work(&work_receivers[0]).1, vec![vec![2], vec![0]]);
     }
+
+    fn create_container_with_capacity(
```

Comment:
can we not share some impl with `create_container` above?

Reply:
Yes. Duplicating code to make this commit easy to read for its intention. Will refactor this function and fold the new "_small" and _large" tests into one.